assets/js/linear_algebra/lib.js

```diff
@@ -618,45 +618,42 @@ function create_segments(d, k=10) {
   return r;
 }
 
+
 function text_table_to_list(texts, start_coord_x, start_coord_y,
                             col_unit, row_unit,
-                            cols_width_array, rows_width_array, // [a, b, ...] means a unit, b unit ...;
-                            start_coord_z=0){
-    var numb_of_rows = texts.length,
-        numb_of_cols = cols_width_array.length + 1,
-        col_coords = [],
-        row_coords = [];
-
-    // setting the collumns' coords:
-
-    col_coords.push(start_coord_x);
-    for (var j = 1; j < numb_of_cols; j++) {
-      col_coords.push(col_coords[j-1] +
-                           col_unit * cols_width_array[j-1]);
-    };
-
-    // setting the rows' coords:
-    row_coords.push(start_coord_y);
-    for (var i = 1; i < numb_of_rows; i++) {
-        row_coords.push(row_coords[i-1] +
-                           row_unit * rows_width_array[i-1]);
-    }
+                            cols_width_array, rows_width_array
```

Comment:
better names: `col_heights` and `row_widths`

Reply:
or shorter names: `dhs` and `dws` (also sound more mathy)

Reply:
done